fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/Utils.kt

```diff
@@ -152,4 +156,81 @@ fun getAvatarSize(secondaryText: String?, tertiaryText: String?): AvatarSize {
         return AvatarSize.Size40
     }
     return AvatarSize.Size56
+}
+
+
+@Composable
+fun fontTypographyForSlicedAvatar(slicedAvatarSize: SlicedAvatarSize): TextStyle {
```

Comment:
can you move this avatar tokens instead

Reply:
but this is not avatar tokens actually
this is not configurable by user but calculated by us based on avatar size
that's why have added it here